lib/bolt/inventory.rb

```diff
@@ -63,6 +62,7 @@ def initialize(data, config = nil, target_vars: {}, target_facts: {}, target_fea
       @target_vars = target_vars
       @target_facts = target_facts
       @target_features = target_features
+      collect_groups
```

Comment:
Should we also remove `collect_groups` call from `setup_inventory` in catalog.rb? Used to build up new inventory from JSON passed to bolt_catalog.  https://github.com/puppetlabs/bolt/blob/a21403650140b54a9b28953170c97e0246065be4/lib/bolt/catalog.rb#L54-L65

Reply:
yup fixed